tensorbay/cli/log.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Implementation of gas log."""
+
+import sys
+from datetime import datetime
+from typing import Dict
+
+import click
+
+from ..client.struct import Commit
+from .tbrn import TBRN
+from .utility import get_dataset_client, get_gas
+
+
+def _generate_commit_log(commit: Commit, oneline: bool) -> str:
```

Comment:
do not use generate as the prefix of the function, which makes reader think this is a generator function